docs/proposals/non-default-namespace-deployment.md

```diff
@@ -0,0 +1,145 @@
+# Proposal: Supporting non default namespace CSI deployment at Openshift 
```

Comment:
Did you try to find examples of other applications who needs privileged access on OpenShift? How do they handle this problem?

Reply:
Metallb solve this problem is similar way: https://blog.oddbit.com/post/2020-09-27-installing-metallb-on-openshif/